app/routes/events_.$eventId.tsx

```diff
@@ -126,3 +126,12 @@ export function ErrorBoundary() {
   const { eventId } = useParams();
   return <div className="error-container">There was an error loading event by the id {eventId}.</div>;
 }
+
+export const meta: MetaFunction<typeof loader> = ({ data }) => {
```

Comment:
NIT: Can we move Metafunction to the top? 🙂 I feel it should be there.

Reply:
Done!